bin/spark-shell

```diff
@@ -29,7 +29,7 @@ esac
 set -o posix
 
 if [ -z "${SPARK_HOME}" ]; then
-  export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
+  source `dirname $0`/find-spark-home
```

Comment:
Many files are changed with this. Do those scripts need to find SPARK_HOME with pip installed Spark? I assume they should be shipped with Spark release?